src/test/java/com/exasol/releaserobot/validation/PlatformValidatorFactoryTest.java

```diff
@@ -19,12 +18,4 @@ void testCreateProjectValidatorGitHub() {
         assertThat(PlatformValidatorFactory.createPlatformValidator(null, platform, null),
                 instanceOf(GitHubPlatformValidator.class));
     }
-
-    @Test
-    void testCreateProjectValidatorUnsupported() {
```

Comment:
what is removed?

Reply:
This test was not stable, I don't have an unsupported enum value anymore, so the test doesn't work. I removed them for now